src/radical/pilot/utils/misc.py

```diff
@@ -88,12 +82,13 @@ def hostip(req=None, black_list=None, pref_list=None, logger=None):
     # Known intefaces in preferred order
     if not pref_list:
         pref_list = [
-            'ipogif0', # Cray's
-            'br0'      # SuperMIC
+            'ipogif0',  # Cray's
+            'ib0',      # IBM (Summit)
+            'br0'       # SuperMIC
```

Comment:
This fix is needed to make the ZMQ bridges accessible from the compute nodes.  This needs to go into some configuration file eventually.